lib/HaskellLib_H.thy

```diff
@@ -15,13 +15,14 @@ imports
   NatBitwise
   "More_Numeral_Type"
   "Monad_WP/WhileLoopRules"
+  "Monad_WP/OptionMonadWP"
```

Comment:
It seems that this has the side effect of `opt_mapE` reintroduced in `elim!` at some point before ArchDetype_AI, and that is breaking RISCV AInvs. It is removed from `elim!` in ArchInvariants_AI for RISCV, but it is reintroduced via an import of MonadicRewrite in SchedContext_AI, it seems (a late import with a FIXME). This needs to be sorted properly at some point, but, for now, I’ll remove `opt_mapE` from `elim!` for  RISCV at the beginning of ArchSchedule_A.

Reply:
Yes, that's fine. It's probably a sign that it shouldn't be `[elim!]` in the first place, but that is an even larger change.